src/services/price/update-price-history.ts

```diff
@@ -41,10 +40,8 @@ export const updatePriceHistory = async (
 
   // skip if it has not been an hour since last update
   try {
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore-error: TODO
-    const diff = moment().diff(moment(last(doc.pair.exchange.price)._id))
-    if (diff < 1000 * 60 * 60) {
+    const lastUpdate: number | undefined = last(doc.pair.exchange.price._id)
+    if (!!lastUpdate && Date.now() - lastUpdate < 1000 * 60 * 60) {
```

Comment:
Just one small optional nit here

```suggestion
    if (!!lastUpdate && Date.now() - lastUpdate < MS_PER_HOUR) {
```